Homework-24

CassieProgram opened the pull request:
browserfactory grid

CassieProgram committed:
Homework-24

browserfactory grid

khanhtestpro commented:
**// Comment from HW team**
Hello,
Your homework has been reviewed.
Great job on creating Homework 24! However, how are you able to run it on multiple browsers such as Chrome, Firefox, and Edge?
I couldn’t run it on my side — are you facing the same issue?

[https://loom.com/i/0e06e1beef8e4932ac1dcc1ac6ca0671](https://loom.com/i/0e06e1beef8e4932ac1dcc1ac6ca0671)

Best wishes,
The HW Team (Khanh – [[khanh@testpro.io](mailto:khanh@testpro.io)](mailto:khanh@testpro.io))

CassieProgram commented:
Hello.

I can install firefox but not edge and test it again. Homework says just chrome. My computer I am afraid will run out of space ...
________________________________
From: khanhtestpro ***@***.***>
Sent: Wednesday, October 22, 2025 4:44 AM
To: CassieProgram/A75-QA-Automation-Framework1 ***@***.***>
Cc: Shynar Largess ***@***.***>; Author ***@***.***>
Subject: Re: [CassieProgram/A75-QA-Automation-Framework1] Homework-24 (PR #12)

[https://avatars.githubusercontent.com/u/144403954?s=20&v=4]khanhtestpro left a comment (CassieProgram/A75-QA-Automation-Framework1#12)<https://github.com/CassieProgram/A75-QA-Automation-Framework1/pull/12#issuecomment-3431136232>

// Comment from HW team
Hello,
Your homework has been reviewed.
Great job on creating Homework 24! However, how are you able to run it on multiple browsers such as Chrome, Firefox, and Edge?
I couldn’t run it on my side — are you facing the same issue?

https://loom.com/i/0e06e1beef8e4932ac1dcc1ac6ca0671

Best wishes,
The HW Team (Khanh – ***@***.******@***.***>***@***.***))

—
Reply to this email directly, view it on GitHub<https://github.com/CassieProgram/A75-QA-Automation-Framework1/pull/12#issuecomment-3431136232>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BXT7WWYIOOPO4KECGSVDO6T3Y47XBAVCNFSM6AAAAACJ3EKRO6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTIMZRGEZTMMRTGI>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

CassieProgram committed:
Homework-24updated

added firefox

CassieProgram commented:
Hi, when I updated  my test and run and passed it and then when to github desktop to commit the changed there was no any pop up window pull request. Why? Or no push in github desktop just fetch . Does my pull stay the same as before I changed the test?
________________________________
From: Shynar Largess ***@***.***>
Sent: Wednesday, October 22, 2025 9:24 AM
To: CassieProgram/A75-QA-Automation-Framework1 ***@***.***>
Subject: Re: [CassieProgram/A75-QA-Automation-Framework1] Homework-24 (PR #12)

Hello.

I can install firefox but not edge and test it again. Homework says just chrome. My computer I am afraid will run out of space ...
________________________________
From: khanhtestpro ***@***.***>
Sent: Wednesday, October 22, 2025 4:44 AM
To: CassieProgram/A75-QA-Automation-Framework1 ***@***.***>
Cc: Shynar Largess ***@***.***>; Author ***@***.***>
Subject: Re: [CassieProgram/A75-QA-Automation-Framework1] Homework-24 (PR #12)

[https://avatars.githubusercontent.com/u/144403954?s=20&v=4]khanhtestpro left a comment (CassieProgram/A75-QA-Automation-Framework1#12)<https://github.com/CassieProgram/A75-QA-Automation-Framework1/pull/12#issuecomment-3431136232>

// Comment from HW team
Hello,
Your homework has been reviewed.
Great job on creating Homework 24! However, how are you able to run it on multiple browsers such as Chrome, Firefox, and Edge?
I couldn’t run it on my side — are you facing the same issue?

https://loom.com/i/0e06e1beef8e4932ac1dcc1ac6ca0671

Best wishes,
The HW Team (Khanh – ***@***.******@***.***>***@***.***))

—
Reply to this email directly, view it on GitHub<https://github.com/CassieProgram/A75-QA-Automation-Framework1/pull/12#issuecomment-3431136232>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BXT7WWYIOOPO4KECGSVDO6T3Y47XBAVCNFSM6AAAAACJ3EKRO6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTIMZRGEZTMMRTGI>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

khanhtestpro commented:
**// Comment from HW team**
Hello,
Your homework has been reviewed.
Great job on creating Homework 24! However, the script still failed when I ran the following command:

```
gradle clean test -Dbrowser=chrome
```

You can check the issue in the picture below:
[https://www.loom.com/i/34adf4875e99431f8851541a25673799](https://www.loom.com/i/34adf4875e99431f8851541a25673799)

Best wishes,
The HW Team (Khanh – [[khanh@testpro.io](mailto:khanh@testpro.io)](mailto:khanh@testpro.io))
